### DISCUSSION TITLE
Sony KD-65X9005 and ChanSort

### DISCUSSION THREAD

**Discussion by ToaStarr** [PredatH0r/ChanSort#367] [Q&A]
Hi,

anyone got experience with ChanSort and the Sony KD-65X9005?

I can export the channel (sdb.xml file) and edit them via ChanSort, but import doesn't work...

**Comment by PredatH0r** [comment]
Can you please send me your original file (saved as .bak) and the modified version?
Please copy it directly off your USB-stick so that in case the stick contains corrupted data I will see that data and not a prestine copy from your PC.
Also, please make sure to use a stick with FAT32 formatting. Many TVs fail to properly read or write on sticks with NTFS formatting (it randomly works or doesn't)
